ansible/roles/kafka/tasks/deploy.yml

```diff
@@ -15,12 +15,17 @@
     ports:
       - "{{ zookeeper.port }}:2181"
 
+# this is important but do not yet know why
+- name: wait before checking if zookeeper is up
+  pause:
+    seconds: 2
```

Comment:
we will need to dig into the kafka logs and zoookeeper life cycle - likely the isalive check is wrong.